glossary.yml

```diff
@@ -40,14 +39,10 @@
   ar:
     term: "القاعدة ٩٩,٧-٩٥-٦٨"
     def: >
-        يُعبِر عن حقيقة أن ٦٨٪ من القيم تقع ضمن وحدة واحدة من
-        [الانحراف المعياري](#standard_deviation)
-        من
-        [الوسط](#mean)
-        ، بينما ٩٥٪ تقع ضمن وحدتين،
-        و٩٩,٧٪ تقع ضمن ثلاث وحدات
-        بالمقابل ، حوالي ٠,٣ ٪  من القيم تقع  في أكثر من ثلاثة وحدات
-        من الانحرافات المعيارية أعلى أو أقل من المتوسط في معظم الحالات.
+      يُعبِر عن حقيقة أن ٦٨٪ من القيم تقع ضمن وحدة واحدة من [الانحراف
```

Comment:
I think your text editor is changing the order of the words and breaking the links here.